pkg/ingester/ingester_v2.go

```diff
@@ -277,6 +369,12 @@ func newTSDBState(bucketClient objstore.Bucket, registerer prometheus.Registerer
 			Help:    "The total time it takes to purge the TSDB series reference cache for a single tenant.",
 			Buckets: prometheus.DefBuckets,
 		}),
+
+		idleTsdbChecks: promauto.With(registerer).NewCounter(prometheus.CounterOpts{
```

Comment:
I'm not sure this metric is worth it. It looks redundant to me, b
ecause it's equal to `sum(cortex_ingester_idle_tsdb_check_results_total)`.

Reply:
If you end up removing it, then we could rename `cortex_ingester_idle_tsdb_check_results_total` into `cortex_ingester_idle_tsdb_checks_total`.

Reply:
That's true.